<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlackEnrollBatchDlg</class>
 <widget class="QDialog" name="BlackEnrollBatchDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enroll Batch</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widToolBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="viewBlackBatchList"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblBlackInfoFace">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionSelectFiles">
   <property name="icon">
    <iconset resource="mainResource.qrc">
     <normaloff>:/images/file.png</normaloff>:/images/file.png</iconset>
   </property>
   <property name="text">
    <string>Add Files</string>
   </property>
  </action>
  <action name="actionSelectFolders">
   <property name="icon">
    <iconset resource="mainResource.qrc">
     <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
   </property>
   <property name="text">
    <string>Add Folder</string>
   </property>
  </action>
  <action name="actionEditBlackBatch">
   <property name="icon">
    <iconset resource="mainResource.qrc">
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionDeleteBlackBatch">
   <property name="icon">
    <iconset resource="mainResource.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionProcessBlackBatch">
   <property name="icon">
    <iconset resource="mainResource.qrc">
     <normaloff>:/images/process.png</normaloff>:/images/process.png</iconset>
   </property>
   <property name="text">
    <string>Process</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mainResource.qrc"/>
 </resources>
 <connections/>
</ui>
